chemprop/nn/loss.py

```diff
@@ -152,19 +150,25 @@ def forward(self, preds: Tensor, targets: Tensor, *args) -> Tensor:
 
         return L_nll + self.v_kl * (L_reg - self.eps)
 
-    def get_params(self) -> list[tuple[str, float]]:
-        return [("v_kl", self.v_kl), ("eps", self.eps)]
+    def extra_repr(self) -> str:
+        if self.v_kl != 0.2 and self.eps != 1e-8:
+            return f"v_kl={self.v_kl}, eps={self.eps}"
+        elif self.v_kl != 0.2:
+            return f"v_kl={self.v_kl}"
+        elif self.eps != 1e-8:
+            return f"eps={self.eps}"
+        return ""
```

Comment:
```suggestion
       return f"v_kl={self.v_kl}, eps={self.eps}"
```
Omitting default values is sklearn style. torch style `repr()` includes them:
```python
>>> nn.Dropout()
Dropout(p=0.5)
```

Reply:
Good job torch, I preferred not omitting defaults, so will commit these.